### DISCUSSION TITLE
Is there a way to create a Query Cube like the one provided by Java api?

### DISCUSSION THREAD

**Discussion by amhatre** [atoti/atoti#496] [Q&A] [answered]
Hello,
Using the Java api of Atoti+ we can create a Query Cube that provides a view across the data cubes, as explained [here](https://activeviam.com/activepivot/latest/docs/distributed/distributed_intro/#query-cubes). Is similar feature available in the Atoti+ Python api?

**Comment by bloussou** [comment, marked as answer]
Hello,
Are you in a distributed cluster or do you want to query a remote ActivePivot server? If it is the second part you can create a a [`QuerySession`](https://docs.atoti.io/latest/lib/atoti-query/atoti_query.query_session.html#atoti_query.query_session.QuerySession) and see the cubes using `session.cubes`.

**Reply by amhatre** [reply]
I have 3 cubes created using [session.create_cube()](https://docs.atoti.io/latest/lib/atoti.session.html#atoti.session.Session.create_cube). Each cube is at a different granularity - Trade, Position and Fund level.
I was wondering if I can create a query / combined cube that can provided a union of the 3 data cubes, i.e. polymorphic distribution.

**Reply by bloussou** [reply]
No this is not supported by atoti.

**Reply by amhatre** [reply]
Thanks for confirming Brieuc!
